README.md

```diff
@@ -77,3 +77,11 @@ efforts to rewriting it from scratch.
 The result of that is https://github.com/jsonnet-libs/k8s and expected to
 replace both, https://github.com/ksonnet/ksonnet-gen and
 https://github.com/kube-jsonnet/k as soon as possible.
+
+#### Where can I find documentation?
+
+[Here](https://jsonnet-libs.github.io/k8s-alpha/)
```

Comment:
Having links say here is considered a bad practice, especially because screen readers have a hard time with it.

Instead I suggest just pasting the raw link, or adding an actual description :)